docs/github.md

```diff
@@ -0,0 +1,68 @@
+For multiple of these actions, you need to get the column_id for parameters. Here are the instructions on how to retrieve them.
+
+## Create personal access token
+
+Create a personal access token (classic) with permissions for:
+- repo
+- workflow
+- user
+- project
+
+[docs](https://docs.github.com/en/enterprise-server@3.9/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+
+### Save it
+
+Save it in your shell like this:
+
+```bash
+TOKEN=YOUR_GITHUB_PERSONAL_TOKEN
+```
+
+Or set it as a `Authorization` header in [GraphiQL](https://github.com/graphql/graphiql) like this:
+
+```
+bearer YOUR_GITHUB_PERSONAL_TOKEN
+```
```

Comment:
Can you make it a single copy/paste `curl` command with headers and post content? That should be available on both, windows and UNIX the same way. It would also merge this and the query at the bottom into a single section that will be easy to use and follow